include/blob/blob_client.h

```diff
@@ -400,7 +412,8 @@ namespace azure { namespace storage_lite {
         /// <param name="blob">The blob name.</param>
         /// <param name="is">The source stream.</param>
         /// <param name="metadata">A <see cref="std::vector"> that respresents metadatas.</param>
-        void upload_block_blob_from_stream(const std::string &container, const std::string blob, std::istream &is, const std::vector<std::pair<std::string, std::string>> &metadata = std::vector<std::pair<std::string, std::string>>());
+        /// <param name="streamlen">Length of the stream. Used only when the stream does not support tellg/seekg</param>
+        void upload_block_blob_from_stream(const std::string &container, const std::string blob, std::istream &is, const std::vector<std::pair<std::string, std::string>> &metadata = std::vector<std::pair<std::string, std::string>>(), unsigned int streamlen=0);
```

Comment:
>unsigned int [](start = 234, length = 12)

Other parts for this SDK is using `unsigned long long` as the type of length. This should be aligned.